### ISSUE TITLE
a11y: typst compile fails with --pdf-standard=ua-1

### ISSUE THREAD

**Issue by gordonwoodhull** [flavio20002/typst-orange-template#38]
Hi @flavio20002!

Quarto just added PDF Standards and Accessibility in 1.9 and I ran all our tests with UA-1 accessibility enabled with Typst 0.14 `--pdf-standard=ua-1`

typst-orange-failed with the error below.

It looks like there is a structural problem here; I haven't investigated yet but will be glad to contribute. 

Most of the other failures we see for Typst are expected, like missing alt text and missing title.

The one for orange-book is more interesting...

```
% typst compile --root . --pdf-standard=ua-1 example/main.typ
error: PDF/UA-1 error: invalid document structure, this element's PDF tag would be split up
   ┌─ my-outline.typ:10:2
   │  
10 │ ╭   box(width: 100%, inset: (y: insetSize))[
11 │ │     #grid(
12 │ │       columns: (1.2cm, 1fr, auto),
13 │ │       align: (left+top, left, left),
   · │
24 │ │     )
25 │ │   ]
   │ ╰───^
   │  
   = hint: this is probably caused by paragraph grouping
   = hint: maybe you've used a `parbreak`, `colbreak`, or `pagebreak`
```
